src/Chapter04/Listing04.49.InputValidation.cs

```diff
@@ -31,7 +31,7 @@ public static void Main(params string[] args)
             }
             else
             {
-                System.Console.WriteLine(
+                Console.WriteLine(
                   "\nERROR:  Enter a value from 1-9. "
                   + "Push ENTER to quit");
```

Comment:
```suggestion
                  $"{ Environment.NewLine }ERROR:  Enter a value from 1-9. "
                  + "Push ENTER to quit");
```
can we use environment.newline over \n here?